app/assets/stylesheets/components/common/_travel_insurance_wizard_form.scss

```diff
@@ -0,0 +1,20 @@
+.form.js-wizard{
```

Comment:
We prefer not to concatenate classnames but rather to use nesting in this instance. However the use of the `js-wizard` class is not really necessary here since we are using Modernizr to attach a `js` class to the `html` element and our preferred method of defining classes for JS components is to wrap the styles in a `.js` container in the SASS.